package.json

```diff
@@ -3,22 +3,38 @@
   "version": "5.2.2",
   "description": "IIIF 2.1 & 3.0 Image API modules for NodeJS",
   "main": "dist/index.js",
-  "types": "types/index.d.ts",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "exports": {
+    ".": {
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.ts",
+        "default": "./dist/index.js"
+      }
+    }
+  },
   "repository": "https://github.com/samvera/node-iiif",
   "author": "Michael B. Klein",
   "license": "Apache-2.0",
   "files": [
     "CHANGELOG.md",
     "CONTRIBUTING.md",
-    "dist/**/*.js",
-    "types/**/*.d.ts",
+    "dist/**/*",
     "LICENSE",
     "package.json",
     "README.md"
   ],
+  "workspaces": [
+    "examples/tiny-iiif",
+    "examples/tiny-iiif/clover-ui"
+  ],
```

Comment:
On your branch, the typecheck was failing without going into each of these example directories and running `npm i`. By using workspaces, `npm i` at the root installs all these deps.